src/drivers/batt_smbus/batt_smbus.cpp

```diff
@@ -55,7 +56,7 @@ BATT_SMBUS::BATT_SMBUS(I2CSPIBusOption bus_option, const int bus, SMBus *interfa
 	_batt_topic = orb_advertise(ORB_ID(battery_status), &new_report);
 
 	int battsource = 1;
-	param_set(param_find("BAT_SOURCE"), &battsource);
+	param_set((param_t)(px4::params::BAT_SOURCE), &battsource);
```

Comment:
Actually, maybe we should leave this alone until we can take a dedicated look at C++ param usage separately. The `px4::params` enum is really only meant to be used with ModuleParams, however I don't see why we shouldn't use it directly with along with a few other new helpers.

Reply:
@dagar Done